README.md

```diff
@@ -1,21 +1,19 @@
 # InceptionV3 Model in Azure AI Gallery
 
 ## Intro
-This repository contains code which demonstrates how to deploy and use an Inception model in Azure ML.
+This repository contains code which demonstrates how to deploy and use an Inception model in Azure Machine Learning.
 
 ## How to Use
-1. [![Azure Notebooks](https://notebooks.azure.com/launch.png)](https://notebooks.azure.com/import/gh/gogowings/AML-Gallery-InceptionV3)
-[Import sample notebooks ](https://notebooks.azure.com/import/gh/gogowings/AML-Gallery-InceptionV3) into Azure Notebooks.
-2. First, run this [configuration notebook](00.configuration.ipynb) to create your workspace. 
-3. Then, run the [01.deploy-inception-model notebook](01.deploy-inception-model) to deploy inception v3 model to a web service and try out for inference.
+1. Use [![Azure Notebooks](https://notebooks.azure.com/launch.png)] to [import the sample notebooks](https://notebooks.azure.com/import/gh/Azure-Samples/AML-Gallery-InceptionV3) into Azure Notebooks.
```

Comment:
add (https://notebooks.azure.com/import/gh/Azure-Samples/AML-Gallery-InceptionV3) right after [![Azure Notebooks](https://notebooks.azure.com/launch.png)], otherwise clicking the AZ NB badge won't trigger the import.